apps/condo/domains/news/hooks/useNewsItemsAccess.ts

```diff
@@ -13,9 +13,9 @@ export const useNewsItemsAccess = (): TUseNewsItemsAccess => {
 
     const canManage = get(organizationLink, ['role', 'canManageNewsItems'], false)
 
-    const roleNameRaw = get(organizationLink, ['role', 'nameNonLocalized'])
+    // const roleNameRaw = get(organizationLink, ['role', 'nameNonLocalized'])
     // The dispatcher is the only one role who can read and can't manage
-    const canRead = canManage || roleNameRaw === 'employee.role.Dispatcher.name'
+    const canRead = get(organizationLink, ['role', 'canReadNewsItems'], false)
```

Comment:
I think it would be a good idea to extend this functionality to all domains

Reply:
I think we can do something in `useOrganization` hook, but in other pr